map/lanelet2_extension/lib/query.cpp

```diff
@@ -87,7 +87,14 @@ lanelet::ConstLanelets query::walkwayLanelets(const lanelet::ConstLanelets lls)
 
 lanelet::ConstLanelets query::roadLanelets(const lanelet::ConstLanelets lls)
 {
-  return query::subtypeLanelets(lls, lanelet::AttributeValueString::Road);
+  lanelet::ConstLanelets road_lanelets;
+  road_lanelets = query::subtypeLanelets(lls, lanelet::AttributeValueString::Road);
+  if (road_lanelets.empty()) {
+    RCLCPP_WARN_STREAM(
+      rclcpp::get_logger("lanelet2_extension.query"),
+      "No road found in lanelet map. Thus vector map will not appear on Rviz.");
```

Comment:
I think this error message should be written in the visualizer package because this is a common library.

Reply:
Exactly, thank you.
I fixed it by https://github.com/autowarefoundation/autoware.universe/pull/1218/commits/63190ebc65d767791f9ddaf6cb3d1a6c85519bbc
and I checked it works fine like
```
[component_container-7] [WARN 1656862383.497976160] [lanelet2_extension.query]: No road found in lanelet map. Thus vector map will not appear on Rviz.
```